React/Base/RCTTouchHandler.m

```diff
@@ -150,6 +150,11 @@ - (void)_updateReactTouchAtIndex:(NSInteger)touchIndex
   reactTouch[@"locationX"] = @(touchViewLocation.x);
   reactTouch[@"locationY"] = @(touchViewLocation.y);
   reactTouch[@"timestamp"] =  @(nativeTouch.timestamp * 1000); // in ms, for JS
+  
+  if ([UIDevice currentDevice].systemVersion.floatValue >= 9) {
```

Comment:
I agree that check needs to exist... but I think dev should do that in-app (using another API). as 99% of the time ( i assume) dev would need to also alter or disable the feature. 
As it is written, the values are automatically null if the user has iOS 9 but no force touch / not enabled